index.js

```diff
@@ -1,18 +1,233 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
-}
+//Needed to form all the promises you see below
+
+const readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+// Essentially my main menu. User should be able to select which game to start
+init();
+async function init() {
+  console.log(
+    "Hello There! Welcome(or Welcome Back XD) to my very first game COLLECTION :D Here you can choose between two different games. I hope you have fun!"
+  );
+  let choice = await ask("Would you like to play Game 1 or Game 2?\n");
+  if (choice === "Game 1" || choice === "game 1") {
+    //The function to start Game 1
+    start();
+  } else if (choice === "Game 2" || choice === "game 2") {
+    //The function to start Game 2
+    start2();
+  }
+  else {console.log("I didn't think the instructions were that hard but...it seems maybe they were. Let's loop it back and have you read one more time")
+init()}
+}
+
+//The range of possible numbers within the game
+let arr = [
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22,
+  23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41,
+  42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60,
+  61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79,
+  80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98,
+  99, 100,
+];
+
+//Generated random CPU number to use as a first guess
+let randomNum = Math.floor(Math.random() * 100 + 1);
+//Game 1

```

Comment:
This is a fun use of an array, but you could make your code more programmatic but creating a `min` and `max` variable and set numbers to them (ie `let min = 1` and `let max = await ask('what would you like the high range to be')` then `max = parseInt(max)` to turn it from a string to a number. Then you can reset these values as you go along, which would allow you to complete the _Modify your guess range_ story by asking if the guess is higher or lower and then reguessing, rather then display a list of possibilities